src/pages/Index.vue

```diff
@@ -23,12 +23,23 @@ q-page(padding)
 <script lang="ts">
 import { defineComponent } from "vue"
 import { globalState as global } from "src/lib/global"
+import { config } from "src/lib/util"
+import { startFarming } from "src/lib/client"
 const lang = global.data.loc.text.index
 
 export default defineComponent({
   data() {
     return { lang }
   },
+  async mounted() {
+    const configData = await config.read()
+    // If the app star and config with plot and account exists, start the farmer 
```

Comment:
as I explained in the above comment, I believe it is fine for now. But maybe we can add a TODO here to redirect into plottingProgress page, and not to show information again.

Or, we can make the mnemonic modal a separate one, and will be before plottingProgress page. That way, we can just redirect to plottingProgress page here, without additional logic

Reply:
Wath if. In case the farmer got 10 days disconnected and the user re open the app, we can keep going to dashboard and update the Plot Status Label with a loader and % progress. In the next branch we already have the data to show it. WDYT?